src/classes/RLLP_OppRollup_UTIL.cls

```diff
@@ -319,67 +317,54 @@ public without sharing class RLLP_OppRollup_UTIL {
     /*** Multicurrency Support Methods ***/
     
     /*********************************************************************************************************
-    * @description Determines if multi-currency is enabled for the org.
+    * @description Determines if multi-currency is enabled for the org, but also adds the currency iso code
+    * fields to the related
     * @return Boolean representing whether or not multi-currency is enabled.
     */
     public static boolean isMultiCurrency(){                    
+
         if (multiCurrency==null){
             //create currency field handles for later dynamic dml
```

Comment:
This class can be refactored further to remove older logic, if time allows and it will help with readability/functionality.